modules/destroy-hc-ibmpower-cli.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-destroy/hcp-destroy-ibmpower.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="destroy-hc-ibmpower-cli_{context}"]
+= Destroying a hosted cluster on {ibm-power-title} by using the CLI
+
+You can use the `hcp` command-line interface (CLI) to destroy a hosted cluster on {ibm-power-title}.
```

Comment:
Explain the "why" before the "how": "To destroy a hosted cluster on {ibm-power-title}, you can use the `hcp` command-line interface (CLI).